backend/src/gamestate/game.rs

```diff
@@ -29,6 +31,56 @@ impl Game {
         }
     }
 
+    pub fn start(&mut self) -> anyhow::Result<()> {
+        if self.status == GameStatus::Running {
+            anyhow::bail!("Attempted to start an already running game.")
+        }
+
+        self.randomize_player_order();
+        self.randomize_starting_player();
+        self.clear_player_positions();
+
+        self.status = GameStatus::Running;
+        self.deal_starting_cards()?; // must be called after clear_players(), of course
+
+        // todo!("Send STATUS WSMessages to all players, don't have the API yet");
+
+        Ok(())
+    }
+
+    fn randomize_player_order(&mut self) {
+        self.players.shuffle(&mut rand::thread_rng())
+    }
+
+    /// Imitates a random starting player by pretending that some rounds have already been played.
+    fn randomize_starting_player(&mut self) {
+        self.turns_played = rand::thread_rng().gen_range(0..self.players.len());
+    }
+
+    fn clear_player_positions(&mut self) {
+        for player in self.players.iter_mut() {
+            player.clear_position();
+        }
+    }
+
+    /// Clears all players' hands and gives them new cards from a new Deck.
+    fn deal_starting_cards(&mut self) -> anyhow::Result<()> {
+        self.deck = Deck::new();
+
+        for player in self.players.iter_mut() {
+            player.drop_all_cards();
+
+            for _ in 0..CARDS_DEALT_AT_GAME_START {
```

Comment:
Originally, these for loops were in the opposite order because of how Cards are usually dealt IRL. Given the proper random shuffling of cards, the current order shouldn't be that big of an issue, and it allows to use one less for-loop, which would have to be
```rust
for player in self.players.iter_mut() {
    player.drop_all_cards();
}
```
followed by
```rust
for _ in 0..CARDS_DEALT_AT_GAME_START {
    for player in self.players.iter_mut() {
        // card dealing
    }
}
```